extensions/multitenancy.md

```diff
@@ -0,0 +1,138 @@
+#Multitenancy Extension
```

Comment:
Since this is a new page, we need to add it to the index of the Reference Guide too.
You can do this by adjusting the `SUMMARY.md` file.